bench/ann/run.py

```diff
@@ -121,7 +125,10 @@ def main():
         search = args.search
 
     # Read configuration file associated to dataset
-    conf_filepath = args.configuration
+    if args.configuration:
+        conf_filepath = args.configuration
+    else:
+        conf_filepath = os.path.join(scripts_path, "conf", f"{args.dataset}.json")
```

Comment:
Since dataset has a default, that enforces that one of them is specified. Should I remove the default and throw an error instead if both are unspecified?